Get conda package version from setup.py.

ceball opened the pull request:
Demo

(will fail because conda version can't have hyphen in it)

ceball committed:
Get conda package version from setup.py.

jlstevens commented on README.md:
```diff
@@ -10,4 +10,4 @@ Autover provides:
      script](https://github.com/ioam/autover/blob/master/scripts/autover).
 
-Authors: Jean-Luc Stevens, Chris Ball and James A. Bednar
+Authors: Jean-Luc Stevens and James A. Bednar
```
Sneaky! ;-p

ceball replied:
I'm sure Jim didn't appreciate having his name so close to a missing comma.

jlstevens replied:
Ah, that is a very good point!

ceball committed:
Use latest conda build.

Currently 3.2.2. Hashes have been dropped from package names where superfluous.

ceball committed:
Build conda package in PR for demo.

ceball commented:
Replaced by #17
